api/response/schema.py

```diff
@@ -140,12 +146,13 @@ def resolve_get_room_response(self, info, **kwargs):
         per_page = kwargs.get('per_page')
         query = Response.get_query(info)
         room_feedback = query.filter_by(room_id=kwargs['room_id'])
-        if room_feedback.count() < 1:
+        active_feedback = room_feedback.filter(ResponseModel.state == "active")
+        if active_feedback.count() < 1:
```

Comment:
please refactor this to say `if not active_feedback`. the feedback variable has a truthy value, so if it finds an `active_feedback` it would be true if not it would be false.

Reply:
Thank you @vic3king for the feedback. Have made the requested changes.

Reply:
@vic3king, I just found out that after filtering, the data is returned in a list, hence the list will always be true upon querying. This is why I used the count method to check for number in the list. Otherwise, the if condition will not be hit after querying for room responses where there is not active response. I have checked the implementation again after our sync. We can have a sync where I can now show the necessity of this count method. Thanks for the feedback.

Reply:
@IssaIan I see, That makes sense. Good work.